Python_Engine/Query/Import.cs

```diff
@@ -36,10 +39,26 @@ public static PyObject Import(string moduleName)
         {
             // check if python is installed
             if (!Query.IsPythonInstalled())
-                throw new Exception($"Cannot import module {moduleName} because no valid version of Python for the BHoM has been found.\n" +
-                    "Try installing Python and the Python_Toolkit using the Compute.InstallPythonToolkit component.\n" +
-                    "If the installation process fails, pleae consider reporting a bug at " +
-                    "https://github.com/BHoM/MachineLearning_Toolkit/issues/new?labels=type%3Abug&template=00_bug.md");
+            {
+                string installMessage = $"Cannot import module {moduleName} because no valid version of Python for the BHoM has been found.\n" +
+                "Do you want to install the Python_Toolkit?";
+                DialogResult confirmResult = MessageBox.Show(installMessage, "Python not found", MessageBoxButtons.YesNoCancel);
+
+                if (confirmResult == DialogResult.Yes)
+                {
+                    Output<bool, List<string>> result = Compute.InstallPythonToolkit(true);
+                    Reflection.Compute.RecordWarning("Python has been installed, please restart the UI");
```

Comment:
Actually there should be no need to restart. If you recompute all should go smooth.

Can you confirm? If so I'll better specify the warning.

Reply:
That is a good point. I tested with the import numpy method, yes a recompute will do the work without restarting grasshopper. 

It will be great if we can update the installation readme as well as this warning message.

Reply:
I think I might put that on the side and raise an issue for that, maybe that can be picked up later?